core/server/data/migrations/versions/2.18/1-restore-settings-from-backup.js

```diff
@@ -0,0 +1,85 @@
+const _ = require('lodash');
+const Promise = require('bluebird');
+const common = require('../../../../lib/common');
+const config = require('../../../../config');
+const fs = require('fs-extra');
+const path = require('path');
+
+module.exports.config = {
+    transaction: true
+};
+
+const backupFileRegex = /ghost.([\d]{4}-[\d]{2}-[\d]{2}).json$/;
```

Comment:
This can stay how it is because old backups will match this style? We need to delete the broken migration as well, and then everything should be good?

Reply:
e.g. after this version, if they have a backup in the old format - it should not have had the broken migration applied to it